_projects/2023/100481433.Rmd

```diff
@@ -0,0 +1,279 @@
+  scale_y_continuous(labels = scales::percent_format(scale = 100), breaks = intervalos_y) +
+  scale_fill_manual(values = colores_democracias) +
+  scale_color_manual(values = colores_democracias) +
+  coord_cartesian(expand=0) +
+  theme_minimal() +
+  theme(
+    axis.text.x = element_text(color="#5b5b5b", size = 10, family = "Lato", angle = 0, hjust = c(0.1, 0.3, 0.5, 0.5, 0.5, 1), margin = margin(t = 0)),
+    axis.ticks.x = element_line(color="#5b5b5b", size = 0.2),
+    axis.ticks.length = unit(0.1, "cm"),
+    legend.position = "none",
+    panel.grid = element_blank(),
+    axis.text.y = element_text(color="#5b5b5b", size = 10, family = "Lato", margin = margin(r = 0))
+  ) +
+  ggtitle("Countries that are democracies and autocracies, World", subtitle = "Political regimes based on the classification and assessment by Skaaning et al. (2015).") +
+  theme(plot.title = element_text(color="#5b5b5b", family = "serif", size = 15, face = "bold", hjust = -0.27),
+        plot.subtitle = element_text(color="#5b5b5b", family = "Lato", size = 9, hjust = -0.24))
+plot
+```
+
+I employed the intervals created during the data organization for both the x-axis and the y-axis in constructing the graphic. Given that the y-axis represents frequencies in percentage, the code was modified to reflect this in the visualization. The year limits were adjusted to fit the dimensions required for displaying custom labels for each democratic type. While the code's limits span from 1789 to 2085, the graphic displays the period from 1789 to 2022 on the x-axis.
+
+To achieve a custom separation between each year on the x-axis, adjustments were made in the theme section, specifically in *axis.text.x*, where the spacing between each year was manually configured in order to be exactly the distance displayed in the graphic of reference.
+
+### Final touches
+
+I generated custom labels with corresponding colors and segments, mirroring the structure observed in the graphic that I aimed to replicate.
+
+```{r}
+leyenda_custom <- data.frame(
+  Democracy_Type = levels(df_long$Democracy_Type),
+  x = rep(2022, length(levels(df_long$Democracy_Type))),
+  y = seq(0, 1, length.out = length(levels(df_long$Democracy_Type))),
+  label = c("Polyarchies",
+  "Electoral
+democracies",
+  "Male 
+democracies",
+  "Exclusive 
+democracies",
+    "Multi-party
+autocracy",
+    "Multi-party
+autocracies
+without elected
+executive",
+    "One-party
+autocracies",
+    "Non-electoral
+autocracies"
+  ),
+  distancia_etiqueta = c(11, 11, 11, 21, 4, 31, 4, 8),
+  distancia_segmento = c(10, 10,  10,  20,  3,  30,  3,  7),
+  angulo_segmento = c(0, 0, 0, 0, 0, 0, 0, 0),  
+  desplazamiento_vertical_segmento = c(0.20, 0.33, 0.338, -0.16, 0.15, 0.09, 0.01, -0.04),
+desplazamiento_vertical_etiqueta = c(0.20, 0.33, 0.338, -0.16, 0.15, 0.09, 0.01, -0.04),
+extra =c(0, 0, 0, 0, 0, 0, 0, 0)
+)
+Democracy_Type_corregido <- rev(leyenda_custom$Democracy_Type)
```

Comment:
Please restyle this chunk, it's hard to read. If you are trying to get newlines, it's better to put `"Electoral\ndemocracies"` instead. Also fix the indentation of the last two options.

It would be better if all variable names are in English, so that all readers can better understand what you are doing.